eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <PackageVersionNet7>7.0.5</PackageVersionNet7>
     <PackageVersionNet6>6.0.$([MSBuild]::Add($([System.Version]::Parse('$(PackageVersionNet7)').Build),11))</PackageVersionNet6>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>6</PreReleaseVersionIteration>
+    <PreReleaseVersionIteration>7</PreReleaseVersionIteration>
```

Comment:
@lewing @steveisok @radical the failure looks related.

I'm having a deja vu. Did we have a similar failure last month? If yes, should we modify the "main bumping process" to make sure the PR includes all the necessary changes that would avoid this failure?

Reply:
The corresponding emsdk update was needed.